pkg/apiserver/apiserver.go

```diff
@@ -100,6 +101,7 @@ func (s *Service) Start(ctx context.Context) error {
 	s.app = fx.New(
 		fx.Logger(utils.NewFxPrinter()),
 		fx.Provide(
+			fx.Supply(featureflag.NewRegistry(s.config.FeatureVersion)),
```

Comment:
I think your `fx.Supply` should be placed outside `fx.Provide`?

Reply:
Yes, thank you! I'll finish the remain issues today.